frontend/src/static/js/main.js

```diff
@@ -155,6 +166,29 @@ window.app = new window.Vue({
       }
     },
 
+    renderAuthorShipTabHash() {
+      const hash = window.hashParams;
+      const info = {};
+      if (hash.tabAuthor) { info.author = hash.tabAuthor; }
+      if (hash.tabLocation) { info.location = hash.tabLocation; }
+      if (hash.since) { info.minDate = hash.since; }
+      if (hash.until) { info.maxDate = hash.until; }
```

Comment:
The `info.minDate` and `info.maxDate` should not be obtained from hash because the code view is fixed (code does not change with the report selection). Should use the min and max date of the report.